packages/graphback-runtime-mongodb/src/MongoDBDataProvider.ts

```diff
@@ -21,6 +21,10 @@ export class MongoDBDataProvider<Type = any, GraphbackContext = any> implements
     this.tableMap = buildModelTableMap(baseType);
     this.collectionName = this.tableMap.tableName;
   }
+  
+  public setBaseType(baseType: GraphQLObjectType<any, any, { [key: string]: any; }>): void {
```

Comment:
I do not think that setter with work in this scenario. 

What graphback is now is just list of the services where each service contains other service (thru inheritance or composition) and then contains data provider.

Reply:
Base type is the only fundamental variable that is needed in every data provider.

What other arguments, model-specific auth etc?

Reply:
Shouldn't thinks like metrics, auth happen in CRUD service anyway?